assets/js/WfParameter.js

```diff
@@ -272,6 +273,17 @@ export default class WfParameter extends React.Component {
           history={this.props.getParamText('reorder-history')}
         />
       )
+    } else if (id_name == 'rename-entries') {
+      return (
+          <RenameEntries
+              loadAll={this.props.getParamText('display-all')}
+              changeLoadAll={(val) => {this.props.setParamText('display-all', val)}}
```

Comment:
Make this a method on `WfParameter`. Otherwise, every re-render of `WfParameter` causes a re-render of `<RenameEntries>`, even when nothing has changed.